README.md

```diff
@@ -3,7 +3,14 @@ Fabricating and running orchestration graphs
 
 ## Set up
 
-...
+- install meteor
+
+- run the commands
+`meteor npm install`
+`meteor npm install --save react react-dom`
```

Comment:
You don't need to run these commands, because --save means that they get written to package.json, and will be installed on meteor npm install in the future.